pkg/controller/controller_broker.go

```diff
@@ -168,14 +185,16 @@ func (c *controller) reconcileServiceBrokerKey(key string) error {
 // processed and should be resubmitted at a later time.
 func (c *controller) reconcileServiceBroker(broker *v1alpha1.ServiceBroker) error {
 	glog.V(4).Infof("Processing ServiceBroker %v", broker.Name)
+	if broker.Spec.RelistDuration == nil {
```

Comment:
I think this TODO is fixed :)

You should be defensive in programming, but some assumptions that you have validated for already are okay.  So for instance, you should ensure that you don't dereference a nil pointer for an optional field.  However, if that optional field must be set based on a validation, a state where that validation fails in code should be handled like an error.